ibm_catalog.json

```diff
@@ -0,0 +1,92 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-icd-elasticsearch",
+        "label": "Elasticsearch",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "reference_architecture",
+          "solution"
+        ],
+        "keywords": [
+          "elasticsearch",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "elasticsearch secure",
+          "database",
+          "nosql"
+        ],
+        "short_description": "Creates and configures an IBM Elasticsearch instance.",
+        "long_description": "This architecture supports creating and configuring an IBM Elasticsearch instance with KMS encryption and CBR rules.",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/blob/main/README.md",
+        "offering_icon_url": "",
```

Comment:
please add the IBM Cloud Elasticsearch icon to a folder called `images` in this repo, and link to it here. I usually grab the logo from the IBM catalog entry for the service, but it seems its missing -> https://cloud.ibm.com/databases/databases-for-elasticsearch/create

Perhaps see if @SirSpidey could help follow up on that?